components/LandingPage/LandingPageHero.tsx

```diff
@@ -0,0 +1,43 @@
+import { Image } from 'components/Image';
+import styles from '../../styles/LandingPageHero.module.scss';
+import { ImageTypes } from '../../types/Base';
+
+type LandingPageHeroProps = {
+    image: ImageTypes;
+    children: React.ReactNode;
+};
+
+export const LandingPageHero = ({ image, children }: LandingPageHeroProps) => {
+    return (
+        <>
+            <svg width="0" height="0" viewBox="0 0 1 1">
```

Comment:
question: i'm curious why are the width and height set to 0?

Reply:
I did this initially to hide the SVG, but as the SVG doesn't contain any renderable content I think it is safe to remove these. There may be some cross browser issues with this however - I've double checked clipPathUnits and it works pretty much everywhere.